src/components/molecules/OcTextInput/OcTextInput.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
```

Comment:
The `clear` cross icon has a weird vertical offset, see screenshot.

<img width="670" alt="Screenshot 2022-01-10 at 22 13 10" src="https://user-images.githubusercontent.com/3532843/148840336-729f5f45-0c63-45bc-bf2a-cae6b2dd920c.png">

Reply:
Good find, thx